src/day8/re_frame/trace.cljs

```diff
@@ -160,11 +161,11 @@
                                      :re-frame.router/fsm-trigger "#fd701e"
                                      nil)}}
                [:td {:style (merge row-style {:cursor "pointer"})
-                     :on-click #(swap! trace-details assoc keyword-id (not (keyword-id @trace-details)))}
-                (if (or (keyword-id @trace-details)
-                        (:showing @trace-details))
-                  "▼"
-                  "▶")]
+                     :on-click (fn [e]
+                                 (when show?
```

Comment:
it's true that this part is not very comprehensive. I will change it to an `if`-statement that sets the value for `[:overrides id]` to false if `show-row?` is true and otherwise negates the value. This is necessary because when all details are expanded and one should be collapsed, the value is `nil` and would turn into `true` which is not the expected behavior (it should close)